cdc/sink/mq/manager/kafka_manager.go

```diff
@@ -161,39 +160,21 @@ func (m *kafkaTopicManager) getMetadataOfTopics() ([]*sarama.TopicMetadata, erro
 // topics have been created.
 // See https://kafka.apache.org/23/javadoc/org/apache/kafka/clients/admin/AdminClient.html
 func (m *kafkaTopicManager) waitUntilTopicVisible(topicName string) error {
+	topics := []string{topicName}
 	err := retry.Do(context.Background(), func() error {
 		start := time.Now()
-		topicMetaList, err := m.admin.DescribeTopics([]string{topicName})
+		meta, err := m.admin.GetTopicsMeta(topics, false)
 		if err != nil {
-			log.Error("Kafka admin client describe topic failed",
-				zap.String("topic", topicName),
+			log.Warn(" topic not found, retry it",
```

Comment:
why change the log level to warn?

Reply:
it's a retryable error, so change it to warn

We should not use `Error` to frequently, it makes logs horrible.